chat-plugins/poll.js

```diff
@@ -5,6 +5,8 @@
 
 'use strict';
 
+const moment = require('moment');
```

Comment:
1. you can't do this without adding this to package.json

2. we already have a duration library in Tools, no reason to bring another in